geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/data/KafkaDataStoreFactory.scala

```diff
@@ -215,6 +235,32 @@ object KafkaDataStoreFactory extends GeoMesaDataStoreInfo with LazyLogging {
     KafkaDataStoreParams.IndexTiers.lookupOpt(params).flatMap(parse).getOrElse(SizeSeparatedBucketIndex.DefaultTiers)
   }
 
+  /**
+   * Parse the dynamic expiry param value into a seq of pairs
+   *
+   * @param params data store params
+   * @return
+   */
+  private [data] def parseDynamicExpiry(params: java.util.Map[String, Serializable]): Seq[(String, Duration)] = {
+    lazy val key = s"Invalid property for parameter '${KafkaDataStoreParams.DynamicCacheExpiry.key}'"
+    KafkaDataStoreParams.DynamicCacheExpiry.lookupOpt(params).toSeq.flatMap { value =>
```

Comment:
Can you just map over the lookupOpt?

Reply:
i wanted to get a Seq out, with Seq.empty essentially replacing None. I don't generally like having Option[Seq] since it's ambiguous what the difference between None and Some(Seq.empty) is...

Reply:
Ah, I wasn't watching the types closely enough.  Carry on!